src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String WHITE = "white";
+    private static final int RADIUS_DEFAULT = 10;
+    private static final int MAX_RANDOM = 25;
+    private static final int ONE = 1;
+    private Random random = new Random();
+    private int randomInt;
+
+    public Figure getDefaultFigure() {
+        return new Circle(RADIUS_DEFAULT, WHITE);
+    }
+
+    public Figure getRandomFigure() {
+        randomInt = random.nextInt(5) + 1;

```

Comment:
this looks weird, just start from 0 in cases instead

Reply:
This is not strange, it's just that in this method I will not issue 0, that's all